denoise/unsupervised-wiener/main.py

```diff
@@ -0,0 +1,35 @@
+from scipy.signal import convolve2d as conv2
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.